### PR TITLE
Parameters inconsistence fixup

### PR DESCRIPTION
**Pull request by gmrukwa**

### CODE REVIEW

**Comment by gmrukwa** [comment]
[DiviK fails to launch](https://trello.com/c/fzYlwiqg/421-divik-fails-to-launch)

**Comment by gmrukwa** [comment]
[DiviK results endpoint is not implemented yet](https://trello.com/c/hxNzNRIZ/423-divik-results-endpoint-is-not-implemented-yet)

**Comment by gmrukwa** [comment]
[Re-style divik inputs form](https://trello.com/c/lV7a9dVe/415-re-style-divik-inputs-form)

File: .travis.yml [hunk 1]
```diff
@@ -15,17 +17,34 @@ install:
...
           docker push spectreteam/divik-worker:beta
-          docker push spectreteam/divik-worker:$VERSION-beta
+          docker push spectreteam/divik-worker:$MAJOR.$MINOR.$FIXUP-beta
+          docker push spectreteam/divik-worker:$MAJOR.$MINOR-beta
+          docker push spectreteam/divik-worker:$MAJOR-beta
       fi
-      docker push spectreteam/divik-worker:$VERSION-alpha
+      docker push spectreteam/divik-worker:$MAJOR.$MINOR.$FIXUP-alpha
+      docker push spectreteam/divik-worker:$MAJOR.$MINOR-alpha
+      docker push spectreteam/divik-worker:$MAJOR-alpha
```
Comment (mg6) [inline .travis.yml]: 1. You may substitute `spectreteam/divik-worker` with `$TRAVIS_REPO_SLUG` in the entire script.
2. This Docker image tagging and push operations is a lot of duplication. Could you introduce a `STAGE` variable that will be set to `beta` or `alpha` appropriately? This allows for:

```sh
# always build as latest
docker build -t $TRAVIS_REPO_SLUG:latest

# set up stage
if [ $TRAVIS_BRANCH = "master" ]; then
    STAGE=
elif [ $TRAVIS_BRANCH = "develop" ]; then
    STAGE=beta
else
    STAGE=alpha
fi

# retag appropriately
docker tag $TRAVIS_REPO_SLUG:latest $TRAVIS_REPO_SLUG:$MAJOR.$MINOR.$FIXUP${STAGE:+-$STAGE}
docker tag $TRAVIS_REPO_SLUG:latest $TRAVIS_REPO_SLUG:$MAJOR.$MINOR${STAGE:+-$STAGE}
docker tag $TRAVIS_REPO_SLUG:latest $TRAVIS_REPO_SLUG:$MAJOR${STAGE:+-$STAGE}

# distribute
if [ $TRAVIS_BRANCH = "master" ]; then
  docker push $TRAVIS_REPO_SLUG:latest
fi
docker push $TRAVIS_REPO_SLUG:$MAJOR.$MINOR.$FIXUP${STAGE:+-$STAGE}
docker push $TRAVIS_REPO_SLUG:$MAJOR.$MINOR${STAGE:+-$STAGE}
docker push $TRAVIS_REPO_SLUG:$MAJOR${STAGE:+-$STAGE}
```

**Review by mg6 — CHANGES_REQUESTED** [review]
Some Docker-related remarks, otherwise fine.

File: .travis.yml (hunk 1 above)
Comment (gmrukwa) [reply .travis.yml]: Accepted, in slightly modified form.

**Review by mg6 — APPROVED** [review]

**Review by Daroslaw — APPROVED** [review]

**Pull request merged**